report/sections/Chapter 06 - Test runner/Chapter 6.tex

```diff
@@ -1 +1,57 @@
-\chapter{Test Runner} \label{chap:TestRunner}
\ No newline at end of file
+\chapter{Test Runner} \label{chap:TestRunner}
+One of the primary goals of this application to give the professor a way to specificy what constitutes a correct problem solution.
+As previously mentioned in chapter \ref{section:xxx} \todo{Fix ref. Probably write this earlier. Talk reader through decision of specifying criteria->automated checking for fast feedback->test runner}, a common approach to this is the use of test cases.
+
+\section{Cilent-server interaction}
+When a teacher creates a new exercise, a modal prompts them to enter details about the exercise.
+These details include the name of the exercise, instructions for the student, an optional code template to help the student get started, as well as test code.
```

Comment:
This seems like it should be written somewhere else. Maybe this should be a ref to a section where we describe how we satisfy the usecase.

Reply:
We'll move it later